include/ignition/sensors/ImuSensor.hh

```diff
@@ -35,6 +35,17 @@ namespace ignition
   {
     // Inline bracket to help doxygen filtering.
     inline namespace IGNITION_SENSORS_VERSION_NAMESPACE {
+    //
+    // \brief Reference frames enum
+    enum class IGNITION_SENSORS_VISIBLE WorldFrameEnumType
+    {
+      NONE = 0,
```

Comment:
I think should document what each of these entries mean. For ENU, we could have something like the following:

~~~
// East-North-Up: x - East, y - North, z - Up
~~~

from the name alone, it's unclear to me what `NONE` means. Does it mean that we aren't using an offset at all from the world orientation? What happens if you call `SetWorldFrameOrientation(rot, NONE);`

It would be helpful to clarify what this means and to add tests for the cases involving `NONE`

Reply:
well in either case, I think we should still add documentation for each `enum` value, including `NONE`.

if no localization value is specified, then should it call `SetOrientationReference` with an identity Quaternion?

Reply:
Right. I'll add a test case where the ``localization`` tag is empty.

Reply:
the rest of the test cases look good to me, so I think we just need this test case in which the `localization` tag is empty